docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -9,6 +9,7 @@
 using System.Threading.Tasks;
 using Microsoft.ML;
 using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
```

Comment:
please revert this file and instead add a new sample if needed.

Reply:
Reverted.